src/Metadata.aspx.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace GroupDocs.Metadata.WebForms
+{
+    public partial class Metadata : System.Web.UI.Page
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['partial' is gratuitous in this context.](https://app.codacy.com/gh/groupdocs-metadata/GroupDocs.Metadata-for-.NET-WebForms/pullRequest?prid=5102156)